AirCasting/SessionViews/MultiSliderView.swift

```diff
@@ -6,19 +6,24 @@
 //
 
 import SwiftUI
+import Resolver
 
 struct ThresholdsSliderView: View {
     
     @ObservedObject var threshold: SensorThreshold
+    var selectedStream: MeasurementStreamEntity?
     
     var body: some View {
-        MultiSliderView(thresholds: threshold.rawThresholdsBinding)
+        if let selectedStream = selectedStream {
+            MultiSliderView(thresholds: threshold.rawThresholdsBinding, selectedStream: selectedStream)
+        }
     }
 }
 
 struct MultiSliderView: View {
-    
+    @InjectedObject private var userSettings: UserSettings
     @Binding var thresholds: [Float]
+    var selectedStream: MeasurementStreamEntity
```

Comment:
same as above - let's not pull big dependencies into simple Views. They should stay simple and they should not perform this kind of logic.